oaknational/Oak-Web-Application
# [PUPIL-829] Signpost teachers landing in pupil subjects area

## Description

Music year: {owa_music_year}

[Notion ticket here](https://www.notion.so/oaknationalacademy/Signpost-Teachers-landing-in-the-Pupil-area-of-Oak-af01e7c264f947968ac98dc2f1d9c2d6)

Created new component for the teacher signposting within the pupil browse journey.

The component uses document referrer to determine if it is a direct or internal link to the page.

As the component can only check the document referrer after rendering, it also needed to be animated in to reduce the distraction within the page. (this was signed off by Mike)

The component is present on the subjects, options & units page.

## How to test

1. Go to https://deploy-preview-2605--oak-web-application.netlify.thenational.academy/pupils/years/year-1/subjects
2. You should see the banner as you have navigated to it directly

_

1. Go to https://deploy-preview-2605--oak-web-application.netlify.thenational.academy/pupils/years/
2. Select Year 1
3. You should not see the banner as you have navigated to it from another page within the website

_
1. Go to https://deploy-preview-2605--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-10/options
2. You should see the banner as you have navigated to it directly

_
1. Go to https://deploy-preview-2605--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-10-higher/units
2. You should see the banner at the bottom of the page as you have navigated to it directly

The banner should also close using the mouse and using keyboard selection.

## Checklist
